src/test/mochitest/browser_dbg-outline.js

```diff
@@ -9,6 +9,20 @@ function getNthItem(dbg, index) {
   return findElement(dbg, "outlineItem", index);
 }
 
+// Tests that the outline functions for original and pretty printed source matches
+add_task(async function () {
+  const dbg = await initDebugger("doc-scripts.html");
+  const {
+    selectors: { getSelectedSource },
+    getState
+  } = dbg;
+  await selectSource(dbg,"simple1", 1);
+  findElementWithSelector(dbg, ".outline-tab").click();
+  const originalSource = getItems(dbg);
+  findElementWithSelector(dbg, ".prettyPrint").click();
```

Comment:
When writing new tests, it's always a good idea to look for existing tests that do similar things. For example in https://github.com/devtools-html/debugger.html/blob/6c82067dc511efe01186745a7d21fa17e076d920/src/test/mochitest/browser_dbg-tabs-pretty-print.js#L10 we have an example that clicks this button.

This also demonstrates another piece that is missing here. Given that pretty-printing will not happen instantly, there likely needs to be some kind of delay before you continue like the one at https://github.com/devtools-html/debugger.html/blob/6c82067dc511efe01186745a7d21fa17e076d920/src/test/mochitest/browser_dbg-tabs-pretty-print.js#L11